lib/govuk_navigation_helpers/taxon_breadcrumbs.rb

```diff
@@ -10,9 +10,11 @@ def breadcrumbs
       end
 
       ordered_parents << { title: "Home", url: "/" }
+      ordered_parents.reverse!
+      ordered_parents << { title: content_item.title, url: "#content" }
```

Comment:
That's also fine. The reason I think it would be nice to leave it separate for now is to make it explicit that this is new behaviour, but I'm happy either way.